sequence_task_3/student/src/impl.cpp

```diff
@@ -1,2 +1,18 @@
 #include "impl.h"
+#include <sstream>
+#include <iterator>
+#include <numeric>
 
+std::string reverseWords(const std::string& input)
+{
+    std::istringstream iss(input);
+
+    auto res = std::accumulate(std::istream_iterator<std::string>(iss), 
```

Comment:
that's quite interesting, something new :)
please don't use std::accumulate for strings
https://stackoverflow.com/questions/15347123/how-to-construct-a-stdstring-from-a-stdvectorstring